mreg/management/commands/create_citext_extension.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connection
+from sys import stdout
+from django.conf import settings
+from psycopg2 import connect
+
+class Command(BaseCommand):
```

Comment:
This is brilliant, but we should point to some documentation with regards to the dependencies (rpm packages/libraries/etc) for installing CITEXT into a database.

Reply:
Have a look at 121566ee99dc1804942d87f136fe9dab8307624a, that'll point users in the right direction.

Reply:
Right, don't mind me. :)